### ISSUE TITLE
flag emojis vs textual representations of countries

### ISSUE THREAD

**Issue by JohnRDOrazio** [kas-catholic/confessit-web#72]
In the language dropdown, would it be more pleasant to see actual country flags alongside the languages, rather than textual representations of the countries? At least on Windows 11 desktop I'm only seeing textual representations. However it is possible to see flag emojis by using the **Noto Color Emoji** font by Google:

* https://stackoverflow.com/a/70021892 (click on <kbd>run code snippet</kbd> to see it work)
* https://fonts.google.com/noto/specimen/Noto+Color+Emoji

**Comment by kas-catholic** [comment]
I thought we already did this, but I guess it could be broken on Windows...

Here's what it looks like on Firefox/Ubuntu:
![image](https://github.com/kas-catholic/confessit-web/assets/78571006/9a47db29-1605-41fb-9d46-5a5f3743df79)

Could you share a screenshot of what it looks like for you?

It might be that the font it's using on Windows lacks emoji support or something, maybe the font you linked will fix it. I'll play around with this soon and see if I can figure it out.

**Comment by kas-catholic** [comment]
Ah, yes, I think the Stack Overflow you linked is correct about what's happening. I'll try their fix.

**Comment by kas-catholic** [comment]
@JohnRDOrazio Can you let me know if it's fixed on this preview? https://deploy-preview-74--confessit-web.netlify.app/

**Comment by marchiartur** [comment]
> @JohnRDOrazio Can you let me know if it's fixed on this preview? https://deploy-preview-74--confessit-web.netlify.app/

Yes, it's working for me, I'm using Windows 10 and running on Brave (based on Google Chrome)

**Comment by JohnRDOrazio** [comment]
I can also confirm that it's working

**Comment by JohnRDOrazio** [comment]
re-opening since this fix was lost with the latest updates, I've prepared a fix and can open a PR
